### DISCUSSION TITLE
Add: highway=emergency_access_point to micro mapping mode

### DISCUSSION THREAD

**Discussion by RoyalSlash** [Zverik/every_door#618] [Ideas]
I've been using the app to map said points, which serve as reference points for emergencies in the forest.
Unfortunately I wasn't aware that those points don't show up in micro mapping mode which caused me to map duplicates of already existing points.

**Comment by Zverik** [comment]
Duplicate of #587, will also display a warning in such cases with #619. Thanks for reminding me :)
